packages/node-provider/src/types.ts

```diff
@@ -1,6 +1,9 @@
+export type NodeMessageReceivedCallback = (message: NodeMessage) => void;
+
 export interface INodeProvider {
```

Comment:
I think these types should be moved to the CFjs package. Otherwise CFjs is dependent on NodeProvider for types, which is a weird relationship (CFjs should really not depend on any other Counterfactual packages). Plus, these types correspond to the written spec in CFjs (apart from INodeProvider).

Reply:
@mvanderh I'll defer to your best judgement here. These types came to NodeProvider through you :P

Reply:
Yeah I know, different ideas in different times :p